appendix-petsciicodes.tex

```diff
@@ -426,25 +426,25 @@ \section{Escape Sequences}
 Moves the cursor to the last non-whitespace character on the current line.\\
 \hline
 \specialkey{ESC} \megakey{$\uparrow$} &
-Saves the current cursor position. Use \specialkey{ESC} \megakey{$\leftarrow$} (next to \megakey{1}) to move it back to the saved position.\\
+Saves the current cursor position. Use \specialkey{ESC} \megakey{$\leftarrow$} to move it back to the saved position.

```

Comment:
Oh, this line lost its `(next to \megakey{1})` wording, should we keep it? (as the esc-left entry kept its `next to \widekey{RESTORE}`)?